spec/services/coronavirus_pages/model_builder_spec.rb

```diff
@@ -98,5 +70,16 @@
       discard_changes
       expect(coronavirus_page.sub_sections.first.title).to eq live_title
     end
+
+    it "creates sub_sections with a position that reflects their order in the content item" do
+      coronavirus_page
+      discard_changes
+      input_order = live_sections.pluck("title").each_with_index.to_h.invert
+
+      coronavirus_page.sub_sections.each do |sub_section|
+        position = sub_section.position
+        expect(input_order[position]).to eq(sub_section.title)
+      end
+    end
```

Comment:
`coronavirus_page` is defined via `let!` and therefore doesn't need calling directly as it would have already been called as the spec is started.

`input_order` does not rely on the `coronavirus_page` nor `discard_changes` calls. So should really be defined outside the `it` clause.

I'd suggest this:
```ruby
let(:input_order) {  live_sections.pluck("title").each_with_index.to_h.invert }

it "creates sub_sections with a position that reflects their order in the content item" do
  discard_changes
  coronavirus_page.sub_sections.each do |sub_section|
    position = sub_section.position
    expect(input_order[position]).to eq(sub_section.title)
  end
end
```

Reply:
Thanks Rob. I'll remove the unnecessary call of `coronavirus_page` but leave input order where it is, as it's the only place it's used and makes it a bit faster to scan and understand.